packages/blade/src/components/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,54 @@
+import type { ReactElement } from 'react';
+
+import type { IconComponent } from '..';
```

Comment:
Lets be specific here?
```suggestion
import type { IconComponent } from './IconButton.d';
```

Reply:
VSCode auto import 😅